install/jakartaee/bin/ts.jte

```diff
@@ -1287,7 +1287,7 @@ command.testExecuteAppClient2= \
         PATH="${javaee.home.ri}/nativelib" \
         APPCPATH=${ts.home}/lib/tsharness.jar${pathsep}${ts.home}/lib/cts.jar${pathsep}${ts.home}/lib/riinterceptors.jar \
 	TZ=${tz} \
-        ${RI_JAVA_HOME}/bin/java \
```

Comment:
Just to confirm, since interop tests is now removed we wont need the RI_JAVA_HOME correct ?

Reply:
I think that jws + jaxws test folders also use forward/reverse testing with the RI.

Reply:
RI_JAVA_HOME was  used for EJB interop tests, jdk installation separation.

Reply:
@gurunrao Can you suggest if we can retain the RI_JAVA_HOME, default it to JAVA_HOME so the jaxws tests can make use of it, if required.

Reply:
have add back RI_JAVA_HOME, so that in future jaxws tests can make used of them.